pkg/provider/aws/knowledgebase/lambda.go

```diff
@@ -153,6 +154,74 @@ var LambdaKB = knowledgebase.Build(
 			return nil
 		},
 	},
+	knowledgebase.EdgeBuilder[*resources.LambdaFunction, *kubernetes.HelmChart]{
+		Expand: func(lambda *resources.LambdaFunction, destination *kubernetes.HelmChart, dag *core.ResourceGraph, data knowledgebase.EdgeData) error {
+			if len(lambda.Subnets) == 0 {
+				lambda.Subnets = make([]*resources.Subnet, 2)
```

Comment:
I'm still confused why we need this when `len(subparams["Subnets"]) == 2` - I'm sure I'll forget it at some point if it's required.

Reply:
[`MakeSlice`](https://pkg.go.dev/reflect#MakeSlice) is probably what you're looking for